Gemfile

```diff
@@ -51,6 +51,8 @@ unless ENV["PACKAGING"] && ENV["PACKAGING"] == "yes"
     gem "rubocop", "~> 0.46", require: false
     gem "brakeman", require: false
     gem "database_cleaner"
+    gem "pry"
+    gem "pry-nav"
```

Comment:
I don't personally use it, but if you find it useful, go ahead. I don't have any strong opinions on this.